src/applications/personalization/dashboard/components/debts-v2/CopaysCardV2.jsx

```diff
@@ -43,9 +43,9 @@ export const CopaysV2 = ({ copays }) => {
           className="vads-u-font-weight--bold"
           onClick={() =>
             recordEvent({
-              event: 'profile-navigation',
-              'profile-action': 'view-link',
-              'profile-section': 'view-manage-va-copays',
+              event: 'dashboard-navigation',
+              'dashboard-action': 'view-link',
+              'dashboard-section': 'view-manage-va-bills',

```

Comment:
Just to verify, this event name has been cleared/set up with analytics and will show up in GA correct?

Reply:
yep